### ISSUE TITLE
Strings converted incorrectly

### ISSUE THREAD

**Issue by KalebMills** [finnfiddle/words-to-numbers#30]
The current issue we are seeing is `....words 10 a.m` being converted to `10 1.m`.

**Comment by pjryan93** [comment]
I also just ran into this problem. It looks like the letter A is always being converted to 1, which I don't think is correct. I could see it making sense as a feature when fuzzy is enabled. In the example, it is "a hundred" -> 100, when the logic should probably be "hundred" -> 100.

**Comment by KalebMills** [comment]
Yep. We ended up forking the repo to manage our own version. I'm not sure if this is a good assumption for the code base to make, since I think you can still safely assume the outcome they are trying to get, without using `a`

**Comment by csandman** [comment]
Just to let people know, I fixed this issue in my fork: [insomnia-dev/words-to-numbers](https://github.com/insomnia-dev/words-to-numbers)
